libcontainerd/client_local_windows.go

```diff
@@ -326,6 +326,13 @@ func (c *client) createWindows(id string, spec *specs.Spec, runtimeOptions inter
 	}
 	configuration.MappedPipes = mps
 
+	// Add any device assignments
+	if len(spec.Windows.Devices) > 0 {
```

Comment:
Is this length check needed, or will the loop below just do nothing for 0 length?

Reply:
No its not. I removed and just do the iteration.

Reply:
@johnstep - I was wrong :). The length check does make sense with the errors. So I put it back.